pkg/crc/ssh/keys_unix.go

```diff
@@ -38,7 +38,6 @@ func (kp *KeyPair) WriteToFile(privateKeyPath string, publicKeyPath string) erro
 		if err := f.Chmod(0600); err != nil {
 			return err
 		}
-
```

Comment:
Could you also drop this blank line from the first commit?

Reply:
Yup, no problem